css/decanter.css

```diff
@@ -1424,7 +1424,7 @@ a.decanter-button--unstyled {
   .decanter-icon-block .decanter-icon-block__img {
     float: left;
     margin-right: 1rem;
-    margin-right: 1.776889em; }
+    margin-right: 1.5625em; }
```

Comment:
Was this margin change intensional?

Reply:
Yes. Modular scale is being used on this component (it's the only component utilizing this so far) and it has been updated. We can change it use the margin mixin instead (for now), but there is no real visual difference in the style guide and pattern library as it adjusted relatively.

Reply:
As long as this change is acceptable to you 👍